src/components/Widget.tsx

```diff
@@ -39,6 +39,7 @@ export interface WidgetProps extends Flags, TransactionEventHandlers, WidgetEven
   dialog?: HTMLDivElement | null
   className?: string
   onError?: OnError
+  cacheTokens?: boolean
```

Comment:
should we offer the option, or simply cache everything?

In the case where we keep the option for the users, I would add a default value to true. This way anyone can disable if they have reasons for it(¿?), but still works optimized for everyone.

@jainkunal wyt?